projects/robots/festo/robotino3/protos/Robotino3Wheel.proto

```diff
@@ -0,0 +1,87 @@
+#VRML_SIM R2020b utf8
+# license: Apache License 2.0
+# license url: http://www.apache.org/licenses/LICENSE-2.0.
+# documentation url: https://www.cyberbotics.com/doc/guide/robotino3
+# This is a PROTO file for Webots for the robot Robotino 3 from Festo.
+# It is the Robotino 3 robot that can be extended with different features to fullfil your requirement.
+
+# tags: hidden
+# a wheel for Robotino3.proto
+
+PROTO Robotino3Wheel [
+  field SFFloat nr 0
+]
+{
+Transform {
+  rotation 0 0 1 %{=(fields.nr.value*2/3+0.5)*math.pi}%
+  children [
+    Transform {
+      translation 0 0.1826 0.05915
+      children [
+      DEF %{= "WHEEL_"..fields.nr.value }% HingeJoint {
```

Comment:
There is some indentation missing here.

Reply:
Hi Olivier Michel, thanks for your help!

I changed the Robotino3Wheel.proto to # tags: hidden so it theory it should not need any icon file anymore because it is not visible to users. (like kuka/youbot/protos/Arm0Mesh.proto needs no icons file)

I added the FastRobotino.cpp controller code to the original robotino3.c, base.c and base.h controller code.

I changed in robotino3.wbt the ContactProperties values.

On https://cyberbotics.com/doc/guide/robotino3 the ContactProperties should be changed too:

   coulombFriction 0.0,  2.0,  0.0

  <remove frictionRotation>

  <remove forceDependentSlip>

I will soon upload these changes. (except the https page changes)

Reply:
> I changed the Robotino3Wheel.proto to # tags: hidden so it theory it should not need any icon file anymore because it is not visible to users. (like kuka/youbot/protos/Arm0Mesh.proto needs no icons file)

That's right, sorry I overlooked it.

Reply:
> On https://cyberbotics.com/doc/guide/robotino3 the ContactProperties should be changed too:

You should do it in this PR as well, simply modify [this file](https://github.com/cyberbotics/webots/blob/master/docs/guide/robotino3.md). Once merged into the master branch, it will be automatically published on the web site.

Reply:
To save you from clang format errors, you should probably install a clang format linter in your source code editor as explained [here](https://github.com/cyberbotics/webots/wiki/CPP-Coding-Style#tools). With it, your C++ code will be automatically formatted to pass the clang format CI tests.